.github/actions/setup/action.yml

```
@@ -0,0 +1,25 @@
+name: 'setup'
+description: "This action makes creates necessary setup for CI"
```

Never need "this action", "this class", "this function"...

```suggestion
description: "Creates necessary setup for CI"
```